Agreements/Internal/S_D.txt

```diff
@@ -1,29 +1,29 @@
-COPX:EGBB:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:BIG
-COPX:EGBE:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:BIG
-COPX:EGNX:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:BIG
-COPX:EGGD:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:BIG
-COPX:EGVN:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:BIG
-COPX:EGBJ:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:BIG
-COPX:EGDL:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:BIG
-COPX:EGVA:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:BIG
+COPX:EGBB:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:^BIG
+COPX:EGBE:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:^BIG
+COPX:EGNX:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:^BIG
+COPX:EGGD:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:^BIG
+COPX:EGVN:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:^BIG
+COPX:EGBJ:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:^BIG
+COPX:EGDL:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:^BIG
+COPX:EGVA:*:BIG:*:*:London AC Worthing:London AC Dover:23000:*:^BIG
 
-COPX:EGGD:*:*:*:*:London AC Worthing:London AC Dover:23000:*:CPT-BPK
-COPX:EGVN:*:*:*:*:London AC Worthing:London AC Dover:23000:*:CPT-BPK
-COPX:EGBJ:*:*:*:*:London AC Worthing:London AC Dover:23000:*:CPT-BPK
-COPX:EGDL:*:*:*:*:London AC Worthing:London AC Dover:23000:*:CPT-BPK
-COPX:EGVA:*:*:*:*:London AC Worthing:London AC Dover:23000:*:CPT-BPK
+COPX:EGGD:*:*:*:*:London AC Worthing:London AC Dover:23000:*:^BPK
+COPX:EGVN:*:*:*:*:London AC Worthing:London AC Dover:23000:*:^BPK
+COPX:EGBJ:*:*:*:*:London AC Worthing:London AC Dover:23000:*:^BPK
+COPX:EGDL:*:*:*:*:London AC Worthing:London AC Dover:23000:*:^BPK
+COPX:EGVA:*:*:*:*:London AC Worthing:London AC Dover:23000:*:^BPK
 
-COPX:EGFF:*:BIG:*:*:London AC Worthing:London AC Dover:29000:*:BIG
-COPX:EGFH:*:BIG:*:*:London AC Worthing:London AC Dover:29000:*:BIG
-COPX:EGDX:*:BIG:*:*:London AC Worthing:London AC Dover:29000:*:BIG
+COPX:EGFF:*:BIG:*:*:London AC Worthing:London AC Dover:29000:*:^BIG
+COPX:EGFH:*:BIG:*:*:London AC Worthing:London AC Dover:29000:*:^BIG
+COPX:EGDX:*:BIG:*:*:London AC Worthing:London AC Dover:29000:*:^BIG
 
-COPX:EGFF:*:*:*:*:London AC Worthing:London AC Dover:29000:*:CPT-BPK
-COPX:EGFH:*:*:*:*:London AC Worthing:London AC Dover:29000:*:CPT-BPK
-COPX:EGDX:*:*:*:*:London AC Worthing:London AC Dover:29000:*:CPT-BPK
+COPX:EGFF:*:*:*:*:London AC Worthing:London AC Dover:29000:*:^BPK
+COPX:EGFH:*:*:*:*:London AC Worthing:London AC Dover:29000:*:^BPK
+COPX:EGDX:*:*:*:*:London AC Worthing:London AC Dover:29000:*:^BPK
 
-COPX:EGHH:*:SFD:*:*:London AC Worthing:London AC Dover:19000:*:SFD
```

Comment:
Lines 24-29 - not a climbing agreement as far as i can see?

Reply:
Hmmm, it is written as such in the vMATS (that is, without a specified level by point). I can't work out what the level by point would be though... the boundary? But then the aircraft would enter TC SE?

Reply:
Have updated these now